src/Logs.tsx

```diff
@@ -0,0 +1,264 @@
+import React, { useState, useContext, useEffect, ReactElement } from 'react';
+import Calendar from 'react-calendar';
+import 'react-calendar/dist/Calendar.css';
+import moment from 'moment';
+import { Button, Dropdown, Header, Table, Icon } from 'semantic-ui-react';
+import { NumericOnlyInput } from './components';
+import { LogStatsContext, ClickedDayContext } from './Context';
+import { addExerciseToDatabase, retrieveExerciseDatabase, addLogToDatabase, retrieveLogDatabase, removeLogFromDatabase } from './Database';
+import './index.css';
+
+function Logs(): ReactElement {
+  const clickedDay = useContext(ClickedDayContext);
+  const [currentTab, setcurrentTab] = useState(<div />);
```

Comment:
For handling tabs you should install `react-router` and use that instead

Reply:
fixed (?)